localisation/00_zuwc_united_localisation.csv

```diff
@@ -2907,19 +2907,19 @@ song_of_swarm_cooldown;Recent Song of Swarm;;K
 song_of_swarm_modifier;Song of Swarm;;Lied des Schwarms;;;;;;;;;;;x
 song_of_swarm;Song of Swarm;;Lied des Schwarms;;;;;;;;;;;x
 song_of_swarm_desc;You sing to the swarm, giving it the desire to fight, your words filling their minds. Your will is its will, and it won't hesitate to obey.;;Ihr singt zum Schwarm, gebt ihm den Wunsch zu k�mpfen und Eure Worte erf�llen seinen Geist. Euer Wille ist sein Wille und er z�gert nicht zu gehorchen.;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
+has_not_used_decision_tooltip;Has NOT used this decision;;Hat diese Entscheidung NICHT genutzt;;;;;;;;;;;x
+waterspeaking_decision;Waterspeaking;;Wasserrufen;;;;;;;;;;;x
+waterspeaking_decision_desc;Commune with the waters of the river and listen for the future and news it would bring for �Y[Root.GetBestName]�!. It grants one random attribute, trait and may upgrade the education.;;Kommuniziert mit dem Wasser des Flusses und lauscht auf die Zukunft und die Nachrichten, die es f�r �Y[Root.GetBestName]� bringen k�nnte! Es gew�hrt einen zuf�lligen Attributsbonus, eine Eigenschaft und kann die Ausbildung aufwerten.;;;;;;;;;;;x
```

Comment:
Saying `upgrade the education` doesn't really work. I'd change it to your education, or something similar.
```suggestion
waterspeaking_decision_desc;Commune with the waters of the river and listen for the future and news it would bring for �Y[Root.GetBestName]�!. It grants one random attribute, trait and may upgrade your education.;;Kommuniziert mit dem Wasser des Flusses und lauscht auf die Zukunft und die Nachrichten, die es f�r �Y[Root.GetBestName]� bringen k�nnte! Es gew�hrt einen zuf�lligen Attributsbonus, eine Eigenschaft und kann die Ausbildung aufwerten.;;;;;;;;;;;x
```

Reply:
Changed in 99667c2f99f980391703bf1cfd9bae605e417783.